uk_bin_collection/uk_bin_collection/councils/RugbyBoroughCouncil.py

```
@@ -19,129 +12,33 @@ class CouncilClass(AbstractGetBinDataClass):
-
-            soup = BeautifulSoup(driver.page_source, "html.parser")
+        user_uprn = kwargs.get("uprn")
+        check_uprn(user_uprn)
 
-            # Initialize bin data dictionary
-            bin_data = {"bins": []}
+        api_url = (
+            "https://apps.cloud9technologies.com/rugby/citizenmobile/webapi/"
+            f"wastecollections/{user_uprn}"
+        )
+        json_data = requests.get(api_url).json()
```

Ruff (0.15.20)

[error] Probable use of `requests` call without timeout

(S113)